android-components/components/ui/icons/src/main/res/drawable/mozac_ic_shipping_24.xml

```diff
@@ -0,0 +1,16 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+    <path
+        android:fillColor="@color/mozac_ui_icons_fill"
+        android:fillType="evenOdd"
```

Comment:
@gabrielluong isn't `evenodd` causing issues with our vectors?

Reply:
We are getting warnings for them because they aren't supported in < API levels, and there were instances where `evenodd ` weren't actually needed. However, `evenodd` is needed to render certain icons correctly, and there's no way to get around them that we currently know of.

Reply:
The warnings are not there in the app module, guess it's because `vectorDrawables.useSupportLibrary = true` is set in `build.gradle`. Since app is using these icons, it seems fine but maybe we should add this config in the icons module